obspy/scripts/runtests.py

```diff
@@ -45,24 +45,39 @@
     GNU Lesser General Public License, Version 3
     (https://www.gnu.org/copyleft/lesser.html)
 """
+import contextlib
+import os
 import sys
 from pathlib import Path
 
-import pkg_resources
-import obspy
-from obspy.core.util.misc import change_directory
-
 
 # URL to upload json report
 REPORT_URL = "tests.obspy.org"
 
 
+@contextlib.contextmanager
+def change_directory(path):
+    """
+    A context manager to change directory to target path.
+
+    :param path: The path to change to.
+    :type path: A string or pathlib Path.
+    """
+    origin = Path().absolute()
+    try:
+        os.chdir(path)
+        yield
+    finally:
+        os.chdir(origin)
+
+
 def _ensure_tests_requirements_installed():
     """
     Ensure all the tests requirements are installed or raise exception.
 
     This function is intended to help less experienced users run the tests.
     """
+    import pkg_resources
```

Comment:
Did you move this import to speed obspy's overall import speed? I would think obspy.scripts.runtest should be imported unless the test suite is called.